diff --color -Naur librealsense/CMake/realsense2Config.cmake.in librealsense_new/CMake/realsense2Config.cmake.in
--- A/CMake/realsense2Config.cmake.in   2023-01-30 22:35:28.853406445 +0100
+++ B/CMake/realsense2Config.cmake.in   2023-01-30 22:34:57.253404779 +0100
@@ -6,6 +6,8 @@

 set(realsense2_VERSION ${realsense2_VERSION_MAJOR}.${realsense2_VERSION_MINOR}.${realsense2_VERSION_PATCH})

+set(PACKAGE_PREFIX_DIR /usr)
+# set_and_check(realsense2_INCLUDE_DIR "@PACKAGE_CMAKE_INSTALL_INCLUDEDIR@")
 set_and_check(realsense2_INCLUDE_DIR "@PACKAGE_CMAKE_INSTALL_INCLUDEDIR@")

 include("${CMAKE_CURRENT_LIST_DIR}/realsense2Targets.cmake")
